pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml

```diff
@@ -76,8 +76,6 @@ spec:
                 items:
                   type: string
                 type: array
-            required:
```

Comment:
Was this done manually? I don't see a corresponding CRD go code update

Reply:
No. Not manual.

Reply:
Ah, I see. sounds good.